web-client/src/views/CreatePetitionerAccount/CreatePetitionerAccountForm.tsx

```diff
@@ -33,9 +37,15 @@ export const CreatePetitionerAccountForm = connect(
     const [inFocusName, setInFocusName] = useState(true);
     const [submitDisabled, setSubmitDisabled] = useState(false);
 
+    // Re-enabled submit button if submission was unsuccessful
+    useEffect(() => {
+      if (alertError || alertWarning) {
+        setSubmitDisabled(false);
+      }
+    }, [alertError, alertWarning]);
+
     const submitFunction = debounce(() => {
       submitCreatePetitionerAccountFormSequence();
-      setSubmitDisabled(false);
```

Comment:
This was re-enabling before transitioning to the next "Success" page, which was confusing.